vault/token_store.go

```diff
@@ -598,16 +616,25 @@ func (ts *TokenStore) SetExpirationManager(exp *ExpirationManager) {
 	ts.expiration = exp
 }
 
-// SaltID is used to apply a salt and hash to an ID to make sure its not reversible
+// Deprecated: SaltID is used to apply a salt and hash to an ID to make sure
+// its not reversible
 func (ts *TokenStore) SaltID(ctx context.Context, id string) (string, error) {
 	s, err := ts.Salt(ctx)
 	if err != nil {
 		return "", err
 	}
-
 	return s.SaltID(id), nil
 }
 
+func (ts *TokenStore) hmac(ctx context.Context, id string) (string, error) {
```

Comment:
[Optional]: s/hmac/HMAC/

Reply:
Not sure why this is empty, but original comment:

[Optional]: s/hmac/HMAC/.

Reply:
The function is not used outside of token store. So, I did not want to expose it, and it was just easier to keep it lower case.

Reply:
Yeah I was seeing that the expiration manager also calls its token store's `hmac()` func, so I was thinking of maybe treating this as a public method. However, since they are both in the same package it's not a big deal if not.